examples/kubernetes/dynamic_provisioning/specs/storageclass.yaml

```diff
@@ -0,0 +1,12 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: efs-sc
+provisioner: efs.csi.aws.com
+parameters:
+  provisioningMode: efs-ap
+  fileSystemId: fs-92107410
+  directoryPerms: "700"
+  gidRangeStart: "1000"
+  gidRangeEnd: "2000"
+  basePath: "/dynamic_provisioning"
```

Comment:
Can we add comments in the example that these are optional? e.g.

```
gidRangeStart: "1000" # optional
gidRangeEnd: "2000" # optional 
basePath: "/dynamic_provisioning" #optional 
```

Reply:
Updated